exporter/tanzuobservabilityexporter/transformer_test.go

```diff
@@ -308,58 +308,18 @@ func TestSpanForDroppedEventsCount(t *testing.T) {
 	actual, err := transform.Span(span)
 	require.NoError(t, err, "transforming span to wavefront format")
 	assert.NotContains(t, actual.Tags, "otel.dropped_events_count")
+	assert.NotContains(t, actual.Tags, "otel.dropped_links_count")
+	assert.NotContains(t, actual.Tags, "otel.dropped_attributes_count")
 
 	//TestCase2: droppedEvents set
 	span.SetDroppedEventsCount(123)
-
-	actual, err = transform.Span(span)
-	require.NoError(t, err, "transforming span to wavefront format")
-	assert.Equal(t, "123", actual.Tags["otel.dropped_events_count"])
-}
-
-func TestSpanForDroppedLinksCounts(t *testing.T) {
-	inNanos := int64(50000000)
-
-	//TestCase: 1 DroppedLinks tag not set
-	resAttrs := pdata.NewAttributeMap()
-	transform := transformerFromAttributes(resAttrs)
-	span := pdata.NewSpan()
-	span.SetSpanID(pdata.NewSpanID([8]byte{0, 0, 0, 0, 0, 0, 0, 1}))
-	span.SetTraceID(pdata.NewTraceID([16]byte{1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1}))
-	span.SetStartTimestamp(pdata.Timestamp(inNanos))
-
-	actual, err := transform.Span(span)
-	require.NoError(t, err, "transforming span to wavefront format")
-	assert.NotContains(t, actual.Tags, "otel.dropped_links_count")
-
-	//TestCase2: DroppedLinks set
 	span.SetDroppedLinksCount(123)
-
-	actual, err = transform.Span(span)
-	require.NoError(t, err, "transforming span to wavefront format")
-	assert.Equal(t, "123", actual.Tags["otel.dropped_links_count"])
-}
-
-func TestSpanForDroppedAttributesCounts(t *testing.T) {
-	inNanos := int64(50000000)
-
-	//TestCase: 1 DroppedAttributes tag not set
-	resAttrs := pdata.NewAttributeMap()
-	transform := transformerFromAttributes(resAttrs)
-	span := pdata.NewSpan()
-	span.SetSpanID(pdata.NewSpanID([8]byte{0, 0, 0, 0, 0, 0, 0, 1}))
-	span.SetTraceID(pdata.NewTraceID([16]byte{1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1}))
-	span.SetStartTimestamp(pdata.Timestamp(inNanos))
-
-	actual, err := transform.Span(span)
-	require.NoError(t, err, "transforming span to wavefront format")
-	assert.NotContains(t, actual.Tags, "otel.dropped_attributes_count")
-
-	//TestCase2: DroppedAttributes set
 	span.SetDroppedAttributesCount(123)
```

Comment:
Use 789 instead of 123. Using different numbers makes your tests catch certain types of errors such as matching the wrong attribute in span to the wrong tag name.

Reply:
done